docs/extensions-framework/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Extensions framework"
+date: 2020-05-28T10:39:00+01:00
+weight: 90
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs
+geekdocFilePath: extensions-framework/_index.md
+---
+
+Extension framework is one of the fundamental building blocks of oCIS. In the following documentation, you will learn how oCIS extensions work and how to create your own.
```

Comment:
I think we should point more to this documentation in context of explaining how to create the extension rather than to ocis-hello. Of course, we should point to ocis-hello as an example extension which is why it was created.

Pointing to ocis-hello as the place where you learn how to create an extension instead of where you see how it works all together can lead to some confusion IMHO. E.g. in the ocis-hello we use rollupjs to bundle the frontend as amd module. Does this mean we have to use rollup? Why amd? Is it because ownCloud devs prefer that or is it required?

Also, this would lead just to jumping between those two docs. See how to create an ocis extension -> jump into ocis-hello docs. See how to load the extension in ocis-web -> jump back to extensions framework docs.

I'll push a new commit where you can see how I think it with pointing to ocis-hello as an example extension.